src/providers/generic/game/directory_resolver/win/XboxGamePassDirectoryResolver.ts

```diff
@@ -26,7 +26,7 @@ export default class XboxGamePassDirectoryResolver extends GameDirectoryResolver
         } catch (err) {
             return new R2Error(
                 `Unable to resolve the ${game.displayName} install directory`,
-                err.message,
+                `${err}`,
```

Comment:
General pattern is to cast to an `Error` and then pass through the message, so this wasn't inherently wrong. 

What does the output look like if you intentionally throw?

Reply:
The reasoning for this change is that it's not guaranteed that all throws are of the `Error` type (or some of its subclass), given that you can throw anything in JS. This is also reflected in the type checking and you can get warnings about it.

As such it's probably better to simply cast it to a string, at best you'll get output like below, at worst you'll probably see `[object Object]`.

![image](https://user-images.githubusercontent.com/8225825/204618810-540ea3c4-cfd5-4d5c-a929-98a9c7043c5f.png)

Reply:
Cool, just looks like it prepends "Error:", so this is fine